src/common/services/api/designations.service.spec.js

```diff
@@ -90,58 +92,63 @@ describe('designation service', () => {
       frequency: 'NA',
       displayName: 'Steve Peck',
       designationType: 'Staff',
+      orgId: 'STAFF',
       code: '0354433',
       designationNumber: '0354433'
     }
-    it('should get product details for a designation number', () => {
+    it('should get product details for a designation number', done => {
       self.$httpBackend.expectPOST('https://give-stage2.cru.org/cortex/items/crugive/lookups/form?FollowLocation=true&zoom=code,offer:code,definition,definition:options:element:selector:choice,definition:options:element:selector:choice:description,definition:options:element:selector:choice:selectaction,definition:options:element:selector:chosen,definition:options:element:selector:chosen:description',
         { code: '0354433' })
         .respond(200, lookupResponse)
       self.designationsService.productLookup('0354433')
         .subscribe((data) => {
           expect(data).toEqual(expectedResponse)
-        })
+          done()
+        }, done)
       self.$httpBackend.flush()
     })
 
-    it('should get product details for a uri', () => {
+    it('should get product details for a uri', done => {
       self.$httpBackend.expectPOST('https://give-stage2.cru.org/cortex/itemselections/crugive/a5t4fmspmhbkez6cwbnd6mrkla74hdgcupbl4xjb=/options/izzgk4lvmvxgg6i=/values/jzaq=/selector?FollowLocation=true&zoom=code,offer:code,definition,definition:options:element:selector:choice,definition:options:element:selector:choice:description,definition:options:element:selector:choice:selectaction,definition:options:element:selector:chosen,definition:options:element:selector:chosen:description')
         .respond(200, lookupResponse)
       self.designationsService.productLookup('/itemselections/crugive/a5t4fmspmhbkez6cwbnd6mrkla74hdgcupbl4xjb=/options/izzgk4lvmvxgg6i=/values/jzaq=/selector', true)
         .subscribe(data => {
           expect(data).toEqual(expectedResponse)
-        })
+          done()
+        }, done)
       self.$httpBackend.flush()
     })
 
-    it('should handle an empty response', () => {
+    it('should handle an empty response', done => {
       self.$httpBackend.expectPOST('https://give-stage2.cru.org/cortex/itemselections/crugive/a5t4fmspmhbkez6cwbnd6mrkla74hdgcupbl4xjb=/options/izzgk4lvmvxgg6i=/values/jzaq=/selector?FollowLocation=true&zoom=code,offer:code,definition,definition:options:element:selector:choice,definition:options:element:selector:choice:description,definition:options:element:selector:choice:selectaction,definition:options:element:selector:chosen,definition:options:element:selector:chosen:description')
         .respond(200, '')
       self.designationsService.productLookup('/itemselections/crugive/a5t4fmspmhbkez6cwbnd6mrkla74hdgcupbl4xjb=/options/izzgk4lvmvxgg6i=/values/jzaq=/selector', true)
-        .subscribe(() => {
-          fail('success should not have been called')
+        .subscribe(done => {
+          done('success should not have been called')
```

Comment:
Same thing here, calling `done` with an argument fails the test. That's what we want in this case. But I'll switch it back to `fail` since some other tests do it that way.